exonum_client/protoc.py

```diff
@@ -77,9 +78,9 @@ def compile(self, path_in: str, path_out: str, include: str = None) -> None:
         code = protoc_process.wait()
         (_, stderr) = protoc_process.communicate()
         if code == 0:
-            print("Proto files were compiled successfully")
+            logging.info("Proto files were compiled successfully.")
         else:
-            print("Error acquired while compiling files: {}".format(stderr.decode("utf-8")))
+            logging.error("Error acquired while compiling files: {}.".format(stderr.decode("utf-8")))
```

Comment:
I think it worth adding information about what files with what includes were attempted to be compiled as well.